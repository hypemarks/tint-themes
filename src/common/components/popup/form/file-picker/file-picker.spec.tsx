import { render } from '@testing-library/react';
import React from 'react';
import FilePicker from './file-picker';

describe('Test <FilePicker /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(
      <FilePicker values={['value']} id={'id'} onChange={jest.fn} name={'name'} onClick={jest.fn} />
    );
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
