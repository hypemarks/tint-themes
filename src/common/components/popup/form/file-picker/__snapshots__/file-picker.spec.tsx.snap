// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <FilePicker /> Component should render snapshot 1`] = `
<DocumentFragment>
  <input
    id="id"
    name="name"
    style="display: none;"
    type="text"
    value="value"
  />
  <button
    style="color: transparent;"
    type="button"
  >
    <span
      class="file-picker__text-label"
    />
  </button>
</DocumentFragment>
`;
